apps/examples/e2e/tests/test-canvas.spec.ts

```diff
@@ -0,0 +1,169 @@
+import test, { expect } from '@playwright/test'
+import { App, TLGeoShape } from '@tldraw/tldraw'
+import { setup } from '../shared-e2e'
+
+export function sleep(ms: number) {
+	return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+declare const app: App
+
+test.describe('smoke tests', () => {
+	test.beforeEach(setup)
+
+	test('create a shape on the canvas', async ({ page }) => {
```

Comment:
Feels like this test does quite a bit more than just creating a shape, might make sense to break it up?

Reply:
Simplified it!